tests/ops_test.py

```diff
@@ -277,8 +277,8 @@ def test_multi_transform():
     calc1 = nufft1(n_k, c, x)
     calc2 = nufft2(f, x)
     for n in range(n_tr):
-        np.testing.assert_allclose(calc1[:, n], nufft1(n_k, c[:, n], x))
-        np.testing.assert_allclose(calc2[:, n], nufft2(f[:, n], x))
+        np.testing.assert_allclose(calc1[:, n], nufft1(n_k, c[:, n], x), rtol=1e-4)
+        np.testing.assert_allclose(calc2[:, n], nufft2(f[:, n], x), rtol=1e-4)
```

Comment:
Interesting that this is required when openmp is enabled. Do you know if that is expected?

Reply:
I wasn't too surprised, just due to the usual effects of multi-threading changing the order of operations.  Although I put `rtol=1e-4`, I think the biggest difference we actually see is 10x smaller—I just used 1e-4 to avoid spurious test failures in the future.  I did check that the change from multi-threading is much smaller in double precision, like 1e-13.

Reply:
Got it - thanks!